docker-compose.yml

```diff
@@ -59,7 +59,6 @@ services:
     image: node:14
     networks:
       - app
-    user: "node"
```

Comment:
Why remove this?

Reply:
Note need.... the default user is node... only need if you what to run as another user.